devtools/gradle/src/main/java/io/quarkus/gradle/dependency/DependencyUtils.java

```diff
@@ -27,13 +27,7 @@
     private static final String TEST_FIXTURE_SUFFIX = "-test-fixtures";
 
     public static Configuration duplicateConfiguration(Project project, Configuration... toDuplicates) {
-        Configuration configurationCopy = project.getConfigurations().findByName(COPY_CONFIGURATION_NAME);
-        if (configurationCopy != null) {
-            project.getConfigurations().remove(configurationCopy);
-        }
-
-        configurationCopy = project.getConfigurations().create(COPY_CONFIGURATION_NAME);
-
+        Configuration configurationCopy = project.getConfigurations().maybeCreate(COPY_CONFIGURATION_NAME).copy();
```

Comment:
This copy will contains some dependency that may not be enabled right? I think here, we should keep the existing code.

Reply:
We create configuration if it does not exist (maybeCreate) and then copy it. So it should be empty. But I reverted anyway since with old way user can't create "quarkusDependency" configuration and modify this behavior.